celery/app/backends.py

```diff
@@ -57,13 +57,27 @@ def by_name(backend=None, loader=None,
 
 
 def by_url(backend=None, loader=None):
-    """Get backend class by URL."""
+    """Get backend class by URL.
+
+    Return a class object and the url to provide to the backend when it is
+    created.
+    """
+    orig_backend = backend
```

Comment:
No. You care so much about it, you fix it.

Reply:
@josephtate I care about code quality and readability, because it is a maintenance factor. This code is being (and will be) read by many developers, so this is also important for the project.

Reply:
According to the contributor's guide, I shouldn't have to worry about style and code cleanup, so I'm going to let you demonstrate how much you care about this by making the change when or after it's merged.